browser/css/notebookbar.css

```diff
@@ -22,7 +22,7 @@
 	margin: auto 4px;
 }
 
-label.notebookbar,
+label.notebookbar:not(.ui-checkbox-label),
```

Comment:
why do we need this? exceptions make it more complex,
there is no `!important` rule so might be enough to have more detailed selector with rule for checkbox label which will override it